index.html

```diff
@@ -1256,31 +1344,78 @@
   // handle gUM error here
 });
 
+// ... initialize WebGL2 context when application starts.
```

Comment:
pretty useless comment, if people dont know that, they cannot understand any example here :) it is basic knowledge of webgl. Comments should make sense, or dont have them